patch/atmega4809.yaml

```diff
@@ -0,0 +1,5 @@
+_svd: ../svd/atmega4809.svd
+
+_include:
+  - "common/port.yaml"
```

Comment:
Okay, maybe my previous comment was a bit all over the place.  To make it more clear what I meant:  Remove this patch as well, as it also does not really relate to the new AVR MCUs (the 0-series).

Then I will merge this PR and we can start looking into HAL driver implementations.  When things come up during HAL implementation, we can apply patches as needed.  I would not try to get everything right up front as that's not going to be possible anyway.